### PR TITLE
Update document 'Building.md' for net6.0 installed

### PR DESCRIPTION
**Pull request by hwthomas**
@DamianSuess I've recently added this extra section to the [Wiki page](https://github.com/dotdevelop/dotdevelop/wiki/Ubuntu-%28Debian%29) for building Ubuntu.  Are you OK to update your `Building.md` page as well?  

I've noticed recently that Github are going to update their `ubuntu-latest` workflow runner to `22.04 (LTS`).  This shouldn't affect the dotdevelop CI workflow (I think) as we use 20.04 in a container.  However, have you looked at whether dotdevelop will build and run under 22.04 LTS at all?

### CODE REVIEW

**Comment by gluckez** [comment]
@hwthomas it does build and run under 22.04. I created a CI on my own fork a while ago that worked perfectly and work on 22.04 myself.

File: Building.md [hunk 1]
```diff
@@ -94,7 +94,23 @@ mono main/build/bin/MonoDevelop.exe
...
+
+"MSB4236 WorkloadAutoImportPropsLocator could not be found".  
+
+This is described by  [this issue](https://github.com/dotnet/sdk/issues/17461) with the following workaround:-   
+
+Set the environment variable `MSBuildEnableWorkloadResolver=false` prior to starting monodevelop  
+
+eg, in a terminal, before starting dotdevelop as above...   
+  
+    export MSBuildEnableWorkloadResolver=false
```
Comment (lextm) [inline Building.md:112]: The proper way to format command line should be using ``` blocks.

Comment (hwthomas) [reply Building.md:112]: @lextm  Thanks for picking that up.  For multi-line commands, I've tended to be to be a bit lax in differentiating between the 'triple-back-ticks'  method and the 'four leading spaces' approach, and didn't appreciate there was a difference.  I'll update the PR.

**Comment by DamianSuess** [comment]
@hwthomas Sorry for the late response, I've been under the weather the past week. Tis' the season for nasty colds.

Though a minor task, thank you for hopping on this one

**Comment by hwthomas** [comment]
@DamianSuess I've had the same thing here.  It seems to take 3 weeks or more to get over, too!
@gluckez  I managed to build and run OK with linux Mint (Ububtu 22.04), but failed to build the Samsung Net Dbg, with the following error:-  
```
...
[ 42%] Built target corguids
[ 44%] Creating new dotnet project
No usable version of libssl was found
Aborted (core dumped)
gmake[2]: *** [src/CMakeFiles/netcoredbg.dir/build.make:86: src/generrmsg/generrmsg.csproj] Error 134
gmake[1]: *** [CMakeFiles/Makefile2:200: src/CMakeFiles/netcoredbg.dir/all] Error 2
gmake: *** [Makefile:156: all] Error 2
done
```
This is perhaps not surprising, as the `build.sh` script downloads dotnet-sdk-3.1, which expects openssl1.0, whereas ubuntu 22.04 uses openssl3.0.  Have you found a way round this?

**Comment by gluckez** [comment]
@hwthomas yes, I have both openssl1 and openssl3 installed on my system, and when I want to build dotdevelop, I create a symlink to the older version. but maybe it would be better to just use a newer .net sdk?

**Comment by hwthomas** [comment]
@gluckez  @DamianSuess 
On ubuntu 22.04, the latest release (2.2.0-961) of the Samsung NetCoreDbg works fine if the binary files are used.  However, it seems more appropriate to raise a separate issue to deal with this, when it becomes necessary, and just merge this minor update now.   
wdyt?

**Comment by gluckez** [comment]
@hwthomas I don't see why we can't merge this, it looks good to me. I think it could be part of our roadmap to look into making that component .net standard compliant, and get rid of any dependencies that are causing this behaviour. but for that, we need to create a roadmap first.

**Pull request merged**